### ISSUE TITLE
Balancer explainer videos

### ISSUE THREAD

**Issue by jmusighi** [balancer-labs/balancer-bounties#5]
Balancer is one of the more complex protocols in crypto to understand and explain. Content that helps break down these complex concepts into a format that is easy to understand would be a valuable asset.

The project:

A series of short explainer videos (up to 2.5 minutes each). We're open to different formats but we think an [animated](https://www.youtube.com/watch?v=Yx56T202nw0) video or a [whiteboard-style](https://www.youtube.com/watch?v=cizLhxSKrAc) video are good options.

Topics to cover (open to suggestions):

- Balancer overview
- Providing liquidity on Balancer
- Trading on Balancer
- Building on Balancer
- Smart pool overview
- Managed index funds (use case)
- Smart treasuries (use case)
- Liquidity bootstrapping pools (use case)
- Asset managers
- Customizable AMM logic

**Comment by Demosthenes-Eth** [comment]
Quick question regarding this bounty.    Is the bounty for videos for each of those topics?  Or is the bounty for one video out of any of those topics?   Does the bounty apply with each video created or only for the bulk of videos as a whole?

**Comment by jmusighi** [comment]
Hey, sorry for the long delay... This got buried in my inbox.

The bounty amount listed is a rough estimate for the series of videos. So far we have paid between 25- 45 BAL per video for the ones that have been created.

Please let me know if you think you can help us generate high quality content!

**Comment by Demosthenes-Eth** [comment]
@jmusighi all good, completely understand!

I do video production and motion graphic work and also run a boutique motion graphic and animation studio.   Big fan of Balancer and BAL, so when I saw this bounty is definitely piqued my interest.   At that range though, it definitely lends itself more towards simpler video work though if that's what you guys have in mind.   Definitely interested in chatting more and can share some work samples.

**Comment by jmusighi** [comment]
@KW710 I'd love to learn more. Please email me at jeremy at balancer dot finance

**Comment by Jason-Black** [comment]
I agree - I'm interested in this bounty but the reward is too low.  In my commercial clients, I would charge 5k USD for each one of these videos - and that is for midlevel corporate explainer style.  For custom motion gfx, Voice Over, and High Quality, many studios would charge 15k+ for 30 seconds.  Production is an **extremely** labor-intensive process - and you are also asking us to come up with the script as well.  _It would be a lot cheaper if we develop scripts (maybe we get help from the community)_
After that finding, a production company could be a lot easier. (and cheaper)

**Comment by jmusighi** [comment]
@KW710 @Jason-Black 

thanks for the feedback. yes-- originally we were looking for more simple content. 

i'd be happy to learn more about your work and how you would approach creating balancer content. we're open to higher quality stuff.

**Comment by Jason-Black** [comment]
I would approach this by first coming up with core concepts that you want to communicate(will need to coordinate between tech and advertising), then use the community to write a few scripts, then contact a producer (like myself) who you can then pass the project to for final tweaking and animation.
